phys/module_mp_thompson.F

```diff
@@ -419,7 +419,7 @@ SUBROUTINE thompson_init(hgt, orho, nwfa2d, nwfa, nifa,           &
 
 
       INTEGER:: i, j, k, l, m, n
-      REAL:: h_01, airmass, niIN3, niCCN3, max_test
+      REAL:: h_01, airmass, niIN3, niCCN3, max_test, z1
```

Comment:
airmass can also be removed from this code including the later calculation line (466)